### PR TITLE
feat: Add support Linux in ARM64 & Improvement for CI 

### PR DESCRIPTION
**Pull request by flemzord**
# Add support Linux in ARM64

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Technical debt

## What parts of the code are impacted ?
Build process.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by antoinegelloz — DISMISSED** [review]

**Review by gfyrag — DISMISSED** [review]

**Review by devotox — DISMISSED** [review]

**Comment by flemzord** [comment]
@gfyrag & @antoinegelloz now, CI is full green :)

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -98,18 +98,23 @@ jobs:
...
+  GoReleaser:
+    name: 'GoReleaser'
     needs:
       - build_control
       - Test_sqlite
       - Test_postgres
+      - Lint
     runs-on: ubuntu-latest
     steps:
       - uses: actions/setup-go@v2
```
Comment (antoinegelloz) [inline .github/workflows/main.yml:110]: Several Github actions are not up to date, we could maybe use Dependabot?
https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot

File: Taskfile.yaml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+          Image: fedora
+          Cmd: '{{.rpm}} goreleaser-*.armv6hl.rpm'
+      - task: goreleaser:test:pkg
+        vars:
+          Platform: 'arm/7'
+          Image: fedora
+          Cmd: '{{.rpm}} goreleaser-*.armv7hl.rpm'
+
+  goreleaser:test:deb:
+    desc: Tests rpm packages
```
Comment (antoinegelloz) [inline Taskfile.yaml]:
```suggestion
    desc: Tests deb packages
```
?

File: Taskfile.yaml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+          Image: debian
+          Cmd: '{{.dpkg}} goreleaser*_armel.deb'
+      - task: goreleaser:test:pkg
+        vars:
+          Platform: 'arm/7'
+          Image: ubuntu
+          Cmd: '{{.dpkg}} goreleaser*_armhf.deb'
+
+  goreleaser:test:apk:
+    desc: Tests rpm packages
```
Comment (antoinegelloz) [inline Taskfile.yaml]:
```suggestion
    desc: Tests apk packages
```
?

**Review by antoinegelloz — COMMENTED** [review]
Looks good to me. As discussed this morning, it could be a good idea to start using a matrix strategy to run tests for both SQLite and PostgreSQL in a single job definition.

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -215,3 +193,29 @@ jobs:
...
         run: task generate VERSION=develop  -- ${{ matrix.sdk }}
+  Check-pkgs:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_CLI_EXPERIMENTAL: "enabled"
+    needs:
+      - GoReleaser
+    strategy:
+      matrix:
+        format: [ deb, rpm ]
```
Comment (antoinegelloz) [inline .github/workflows/main.yml:204]: Missing apk ?

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -233,3 +209,29 @@ jobs:
...
           commit_user_email: support@numary.com%
+  Check-pkgs:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_CLI_EXPERIMENTAL: "enabled"
+    needs:
+      - GoReleaser
+    strategy:
+      matrix:
+        format: [ deb, rpm ]
```
Comment (antoinegelloz) [inline .github/workflows/release.yml:220]: Missing apk ?

File: .github/workflows/main.yml (hunk 4 above)
Comment (flemzord) [reply .github/workflows/main.yml:204]: Today, We are not yet creating a package for APK, but it should happen

**Review by gfyrag — APPROVED** [review]

**Review by antoinegelloz — APPROVED** [review]

**Pull request merged**